src/components/editor/InputLang.jsx

```diff
@@ -15,47 +14,38 @@ class InputLang extends Component {
     }
   }
 
-  /*
-   * TODO:
-   * English is the default value, but is not set in the redux.lang.store, so it currently would need to be set manually
-   * in the generation of RDF.
-   * See https://github.com/LD4P/sinopia_editor/issues/290
-   */
-
-  /*
-   * When clicking Cancel make it not save the language. Clicking X to remove the input for the literal
-   * leaves the input in the lang redux store but nothing will be associated with it in the "literal" store.
-   * See https://github.com/LD4P/sinopia_editor/issues/275
-   */
-
   setPayLoad(items) {
     const payload = {
-      id: this.props.textValue,
+      id: this.props.textId,
+      reduxPath: this.props.reduxPath,
       items,
     }
 
-    this.props.handleSelectedChange(payload)
+    this.props.handleLangChange(payload)
   }
 
   createOptions = (json) => {
-    for (const i in json) {
-      try {
-        const item = Object.getOwnPropertyDescriptor(json, i)
-        const uri = item.value['@id']
-        const valArr = item.value['http://www.loc.gov/mads/rdf/v1#authoritativeLabel']
-        let label
-
-        valArr.forEach((obj) => {
-          if (obj['@language'] == 'en') {
-            label = obj['@value']
-          }
-        })
-
-        return { id: uri, uri, label }
-      } catch (error) {
-        // ignore
-      }
-    }
+    return json.reduce((result, item) => {
+      // Object.getOwnPropertyDescriptor is necessary to handle the @
+      const uri = Object.getOwnPropertyDescriptor(item, '@id').value
+      const labelArrayDescr = Object.getOwnPropertyDescriptor(item, 'http://www.loc.gov/mads/rdf/v1#authoritativeLabel')
+      // There are some odd entries, so ignoring if don't have labels.
+      if (!labelArrayDescr) return result
+      const labelArray = labelArrayDescr.value
+
+      // Looking for English label
+      let label = null
+      labelArray.forEach((langItem) => {
+        if (langItem['@language'] == 'en') {
```

Comment:
It would be better to get eslint to enforce this for us, but generally I think we should prefer `===` over `==` since the latter does type conversions. (Which can be useful if we want type conversions...)